src/pages/home/report/ReportActionItem.js

```diff
@@ -105,6 +106,8 @@ const defaultProps = {
 
 function ReportActionItem(props) {
     const [isContextMenuActive, setIsContextMenuActive] = useState(ReportActionContextMenu.isActiveReportAction(props.action.reportActionID));
+    const [isHidden, setIsHidden] = useState(false);
+    const [moderationDecision, setModerationDecision] = useState(CONST.MODERATION.MODERATOR_DECISION_APPROVED);
```

Comment:
Hi, we missed a small offline edge case in this PR - https://github.com/Expensify/App/issues/20946

On deleting a hidden message, the reveal message button should have been removed too.